packages/elvis/src/components/form/input.scss

```diff
@@ -290,4 +290,23 @@
       padding: 3px 8px;
     }
   }
+
+  &.e-input---disabled,
+  &:has(input:disabled),
+  &:has(textarea:disabled) {
+    color: var(--e-color-text-disabled-1);
+  }
+
+  &.e-input---readonly {
+    > input,
+    > input:focus,
+    > input:read-only,
+    > input:read-only:focus,
+    > textarea,
+    > textarea:read-only,
+    > textarea:focus,
+    > textarea:read-only:focus {
```

Comment:
Nice to get the readonly as well. We can probably simplify these selectors a bit, using `:is`:
``` css
> :is(input, textarea),
> :is(input, textarea):is(:focus, :read-only, :read-only:focus) {
   ...
}
```